pom.xml

```
@@ -72,30 +72,27 @@
     <project.majorVersion>6</project.majorVersion>
```

```suggestion
    <project.majorVersion>7</project.majorVersion>
```

Should this be updated as well? (Noticed when building the image and it got tagged as `:6`)